tests/integration/targets/s3_sync/tasks/main.yml

```diff
@@ -0,0 +1,116 @@
+---
+- name: set connection information for all tasks
+  set_fact:
+    aws_connection_info: &aws_connection_info
+      aws_access_key: "{{ aws_access_key }}"
+      aws_secret_key: "{{ aws_secret_key }}"
+      security_token: "{{ security_token }}"
+      region: "{{ aws_region }}"
+  no_log: true
+- name: S3 bucket creation
+  collections:
+    - amazon.aws
+    - community.general
+  block:
+    # ============================================================
+    - name: Create simple s3_bucket
+      s3_bucket:
+        name: "{{ resource_prefix }}-testbucket-ansible"
+        state: present
+        <<: *aws_connection_info
+      register: output
+
+    - assert:
+        that:
+          - output.changed
+          - output.name == '{{ resource_prefix }}-testbucket-ansible'
+          - not output.requester_pays
+    # ============================================================
+    - name: Prepare fixtures folder
+      file:
+        path: "{{ output_dir }}/s3_sync"
+        state: directory
+        mode: '0755'
+
+    - name: Prepare files to sync
+      copy:
+        src: "{{ item }}"
+        dest: "{{ output_dir }}/s3_sync/{{ item }}"
+        mode: preserve
+      with_items:
+      - test1.txt
+      - test2.yml
+      - test3.json
+
+    - name: Prepare file with size bigger than chunk size
+      shell: |
+        dd if=/dev/zero of=test4.txt bs=1M count=6
```

Comment:
I'm not sure 6MB is enough:

```
$ python
Python 3.6.8 (default, Jun 12 2019, 01:12:31) 
[GCC 8.2.1 20180905 (Red Hat 8.2.1-3)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from boto3.s3.transfer import TransferConfig
>>> DEFAULT_CHUNK_SIZE = TransferConfig().multipart_chunksize
>>> print(DEFAULT_CHUNK_SIZE)
8388608
>>> 
```

possibly bump this up to 10M ?

Reply:
update the file size for tests to 10Mb